zeppelin-zengine/src/main/java/org/apache/zeppelin/notebook/Note.java

```diff
@@ -472,7 +472,14 @@ public void run(String paragraphId) {
         logger.debug("New paragraph: {}", pText);
         p.setEffectiveText(pText);
       } else {
-        throw new InterpreterException("Interpreter " + requiredReplName + " not found");
+        InterpreterException intpException;
+        intpException = new InterpreterException(
```

Comment:
@bzz 
It made by dividing the problem on the column size limit 100.
I have to fix it again.